scripts/jsonschema_gen.py

```diff
@@ -0,0 +1,109 @@
+try:
```

Comment:
Right now, it does not depend on any of the DVC. The schema is just ~~independent~~ duplicated. But, it cannot be used for validation, at least not yet. :slightly_frowning_face:

Reply:
Same here, if we don't want to maintain it, could push to separate repo.